readme.md

```diff
@@ -162,13 +196,17 @@ Your Rails app has the following `application.html.erb`. Nothing shows up in you
 </html>
 ```
 
-> Your answer...
+> Needs to close the first line and the title tag with a >, i.e. <!DOCTYPE html>
+and <title>Quiz</title>
```

Comment:
:+1: Yes, you are correct, but the more subtle thing is that we need an `=` in the `<%` before `yield` as we want to make sure to print everything inside the block to the screen!